docs/articles/configs/jobs.md

```diff
@@ -55,7 +55,7 @@ In this category, you can specifiy how to benchmark each method.
 * `MinWarmupIterationCount`: Minimum count of warmup iterations that should be performed, the default value is 6
 * `MaxWarmupIterationCount`: Maximum count of warmup iterations that should be performed, the default value is 50
 
-Usually, you shouldn't specify such characteristics like `LaunchCount`, `WarmupCount`, `TargetCount`, or `IterationTime` because BenchmarkDotNet has a smart algorithm to choose these values automatically based on received measurements. You can specify it for testing purposes or when you are damn sure that you know the right characteristics for your benchmark (when you set `TargetCount` = `20` you should understand why `20` is a good value for your case).
+Usually, you shouldn't specify such characteristics like `LaunchCount`, `WarmupCount`, `IterationCount`, or `IterationTime` because BenchmarkDotNet has a smart algorithm to choose these values automatically based on received measurements. You can specify it for testing purposes or when you are damn sure that you know the right characteristics for your benchmark (when you set `TargetCount` = `20` you should understand why `20` is a good value for your case).
```

Comment:
May as well rename `TargetCount` to `IterationCount` everywhere else, too (`TargetCount` = `20`).